scripts/teamd_increase_retry_count.py

```diff
@@ -0,0 +1,236 @@
+                if not interfaceLldpInfo:
+                    print("WARNING: No LLDP info available for {}; skipping".format(portName))
+                    continue
+                interfaceLldpInfo = interfaceLldpInfo[0][portName]
+                peerName = list(interfaceLldpInfo["chassis"].keys())[0]
+                peerInfo = interfaceLldpInfo["chassis"][peerName]
+                if "descr" not in peerInfo:
+                    print("WARNING: No peer description available via LLDP for {}; skipping".format(portName))
+                    continue
+                portChannelChecked = True
+                if "SONiC" not in peerInfo["descr"]:
+                    print("WARNING: Peer device is not a SONiC device; skipping")
+                    failedPortChannels.append(portChannel)
+                    break
+
+                sendReadyEvent = Event()
+
+                # Start sniffing thread
+                lacpThread = LacpPacketListenThread(portName, config["ports"][portName]["runner"]["partner_lacpdu_info"]["system"], sendReadyEvent)
+                lacpThread.start()
+
+                # Generate and send probe packet
+                probePacket = craftLacpPacket(config, portName)
+                sendReadyEvent.wait()
+                sendp(probePacket, iface=portName)
+
+                lacpThread.join()
+
+                resetProbePacket = craftLacpPacket(config, portName, newVersion=False)
+                time.sleep(2)
+                sendp(resetProbePacket, iface=portName, count=2, inter=0.5)
+
+                if lacpThread.detectedNewVersion:
+                    print("SUCCESS: Peer device {} is running version of SONiC with teamd retry count feature".format(peerName))
+                    break
+                else:
+                    print("WARNING: Peer device {} is running version of SONiC without teamd retry count feature".format(peerName))
+                    failedPortChannels.append(portChannel)
+                    break
+            if not portChannelChecked:
+                print("WARNING: No information available about peer device on port channel {}".format(portChannel))
+                failedPortChannels.append(portChannel)
+        if failedPortChannels:
+            print("ERROR: There are port channels/peer devices that failed the probe: {}".format(failedPortChannels))
+            sys.exit(2)
+            return False
+    else:
+        signal.signal(signal.SIGUSR1, abortTeamdChanges)
+        signal.signal(signal.SIGTERM, abortTeamdChanges)
+        retryCountGetProcess = subprocess.run(["config", "portchannel", "retry-count", "get", portChannels[0]])
+        if retryCountGetProcess.returncode == 0:
+            # Currently running on SONiC version with teamd retry count feature
+            for portChannel in portChannels:
+                subprocess.run(["config", "portchannel", "retry-count", "set", portChannel, "5"])
```

Comment:
I suggest to define retry-count 5 and 3 as constants on top of file, and use them here.

Reply:
Constants defined and used.